libs/template/templates/default-sql/databricks_template_schema.json

```diff
@@ -0,0 +1,30 @@
+{
+    "welcome_message": "\nWelcome to the default SQL template for Databricks Asset Bundles!",
+    "properties": {
+        "project_name": {
+            "type": "string",
+            "default": "my_sql_project",
+            "description": "Please provide the following details to tailor the template to your preferences.\n\nUnique name for this project",
+            "order": 1,
+            "pattern": "^[A-Za-z0-9_]+$",
+            "pattern_match_failure_message": "Name must consist of letters, numbers, and underscores."
+        },
+        "workspace_host_override": {
+            "comment": "We explicitly ask users for the workspace_host since we ask for a http_path below. A downside of doing this is that {{user_name}} may not be correct if they pick a different workspace than the one from the current profile.",
```

Comment:
@pietern There's a bit of a trade-off here. {{user_name}} and {{is_service_principals}} don't really work anymore if the user picks a different workspace. But with the question below about the warehouse URL I am leaning towards including this question here. Even if it means that any filled-in user names in the template _might_ be inaccurate. But then if we do it here, perhaps we should also ask about the workspace host in `default-python`?

Reply:
Why do we need the override though? Is the workspace that is selected as part of bundle init not usable?

Reply:
The consideration is really that we ask for a warehouse name. So we basically have three options:
1. We just ask for warehouse id. Then ask for the warehouse id. (Customers might pick a warehouse id from a different workspace and could get really confused.)
2. We first show the workspace URL and don't allow it to be changed. Then ask for warehouse id. (This option might raise the question "but what if I want to deploy elsewhere?).
4. We first show the workspace URL and allow it to be changed. Then ask for the warehouse id. (This option also has downsides as per above :)

So — curious about your thoughts on the trade offs here.

Reply:
All options have the potential to be inconsistent with the selected workspace. IMO, we should either prompt with a dropdown of all warehouses in the workspace or have some kind of live validation.

Until we do, can we assume the selected workspace is the right one and not include the override?

Reply:
Sounds good. Let's go for Option #1 for now.

In the length of time it seems like we should have a native way to ask for the workspace with completion + a native way for selecting the warehouse.